README.md

```diff
@@ -20,77 +20,57 @@
 <!-- TODO: code coverage -->
 </p>
 
-# Overview
-
-Versatile Data Kit (VDK) is an open source framework that enables anyone with basic SQL or Python knowledge to build their own data pipelines.
-
-Versatile Data Kit enables Data Engineers to develop, deploy, run and manage Data Jobs. **A Data Job is a data processing workload** and can be written in Python, SQL, or both at the same time. A Data Job enables Data Engineers to implement automated pull ingestion (E in ELT) and batch data transformation (T in ELT) into a database or any type of data storage.
 
-![Without / With Versatile Data Kit](./support/images/versatile-data-kit-before-after-light.svg#gh-light-mode-only)
-![Without / With Versatile Data Kit](./support/images/versatile-data-kit-before-after-dark.svg#gh-dark-mode-only)
-
-Versatile Data Kit consists of two main components:
+# Overview
+Versatile Data Kit (VDK) is an open source framework that enables anyone with basic SQL or Python knowledge to build and execute their own data workflows.
 
-* A **Data SDK** provides all tools for the automation of data extraction, transformation, and loading, as well as a plugin framework that allows users to extend the framework according to their specific requirements.
-* A **Control Service** allows users to create, deploy, manage and execute Data Jobs in Kubernetes runtime environment.
+Data processing instructions use plain text SQL or python files that are executed sequentially in alphanumeric order, allowing you to easily build your data workflows.
 
-To help solve common data engineering problems Versatile Data Kit:
-* allows ingestion of data from different sources, including CSV files, JSON objects, data provided by REST API services, etc.;
-* ensures data applications are packaged, versioned, and deployed correctly while dealing with credentials, retries, reconnects, etc.;
-* provides built-in monitoring and smart notification capabilities;
-* tracks both code and data modifications and the relations between them, enabling engineers to troubleshoot faster and providing an easy revert to a stable version.
+VDK is built for resiliency and can recover in mid-process or restart entirely from the start.
 
 
-#### Data Journey and where VDK fits in
+# Data Journey and Versatile Data Kit
+VDK creates data processing workflows to:
+- Ingest data (extract)
+- Transform data (transform)
+- Export data (load)
```

Comment:
nit: shoudl we leave an empty line. The picture and the text look strange so close - https://github.com/gary-tai/versatile-data-kit/tree/Update_README#data-journey-and-versatile-data-kit